app/javascript/components/SignIn.jsx

```diff
@@ -65,6 +66,8 @@ export default class SigInForm extends React.Component {
 
           <div>
             <label htmlFor="password">Password</label><br/>
+            <NavLink exact className="nav-link" activeClassName="active" to="/forgot_password"> (Forgot password?)</NavLink>
```

Comment:
maybe it's better to put it before buttons? :) or check some nice looking web app how they do this.